### DISCUSSION TITLE
Adding device | ShellHub

### DISCUSSION THREAD

**Discussion by giscus** [shellhub-io/shellhub#2111] [General]
# Adding device | ShellHub

In order to register a device on ShellHub, you need to install ShellHub agent onto it.

https://docs.shellhub.io/user-guides/devices/adding

**Comment by davidkhala** [comment]
But my container `shellhub` cannot start properly.
It has 

time="2026-08-08T13:52:22Z" level=warning msg="Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?"

**Reply by davidkhala** [reply]
Oh, I guess it does not support rootless docker

**Reply by otavio** [reply]
Yes, unfortunately, it does not. Even if you use Podman, you must run it as root.
